src/app/layout.tsx

```diff
@@ -15,7 +15,7 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={fontJakarta.variable}>
+      <body className={`${fontJakarta.variable} h-screen`}>
```

Comment:
for this there is alternative ways, you can use cn (tailwind merge) so your code will be like this
```tsx
<body className={cn(fontJakarta.variable, "h-screen")}>
```

example code: https://github.com/bandungdevcom/bandungdev.com/blob/staging/app/components/contents/about.tsx

ref: https://medium.com/@nomanfareed681/simplify-your-tailwind-css-class-management-with-merge-and-clsx-42f1e2458fd8